docker/jmeter-base/Dockerfile

```diff
@@ -15,9 +15,7 @@ ENV HEAP -Xms2g -Xmx2g -XX:MaxMetaspaceSize=256m
 RUN apt-get clean \
  && apt-get update \
  && apt-get install --quiet --yes --no-install-recommends \
-    curl=7.74.0-1.3+b1 \
-    vim=2:8.2.2434-3 \
-    procps=2:3.3.17-5 \
```

Comment:
no issue using different version?

Reply:
Builds are failing because these packages are no longer available in the apt repository

Reply:
They should be fine since it was added due to suggestion from `hadolint`, not a specific constraint which should be noted down.